package.json

```diff
@@ -547,6 +578,7 @@
         "@vscode/vsce": "^2.17.0",
         "eslint": "^7.25.0",
         "eslint-plugin-import": "^2.22.1",
+        "fs-extra": "^11.1.0",
```

Comment:
Can we use VSCode's FS instead (or, the one in Utils that uses VSCode's)?

Reply:
It's actually not even used. Not really why I ever installed this as a devDependency at any point